packages/odata-render-engine/yarn.lock

```diff
@@ -2,5287 +2,3 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.0.0-beta.53":
```

Comment:
This file should be deleted?

Reply:
I don't know. We don't have this plugin in package.json, so probably when I added new plugins and presets then yarn automatically removed unnecessary packages.

Reply:
Yes, I removed all dependencies. This package has only peerDependencies, so lock will be always empty :P